src/main/java/net/machpi/runelite/influxdb/MeasurementCreator.java

```diff
@@ -207,4 +211,31 @@ public Series createKillCountSeries(String boss) {
     }
 
     private static final int THRESHOLD = 50_000;
+
+    public Optional<Series> createActivitySeries(ActivityState.State lastState) {
+        Series.SeriesBuilder seriesB = createSeries().measurement(SERIES_ACTIVITY);
```

Comment:
Generally you want to keep the cardinality of your time series database pretty low, so having all these as part of tags is not ideal.